members/apoorva/data.json

```diff
@@ -0,0 +1,13 @@
+{
+	"id": "apoorva",
+	"first_name": "Apoorva",
+	"last_name": "Srivastava",
+	"yoe": 1,
+	"company": "Publicis Sapient",
+	"designation": "Associate Experience Technology L2",
+	"img": "./img.png",
+	"github_id": "Apoorva2696",
+	"linkedin_id": "apoorva-srivastava",
+	"twitter_id": "apoorvasrivastava",
+	"instagram_id": ""
```

Comment:
Line number 12 can be removed if you are not providing the value. Thank you!

Reply:
Done